projects/ngx-maplibre-gl/src/lib/popup/popup.component.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { of } from 'rxjs';
+import { MapService } from '../map/map.service';
+import { MarkerComponent } from '../marker/marker.component';
+import { PopupComponent } from './popup.component';
+
+const getMapServiceStub = () =>
+  jasmine.createSpyObj(
+    [
+      'addMarker',
+      'removeMarker',
+      'removePopupFromMap',
+      'createPopup',
+      'addPopupToMarker'
+    ],
+    {
+      mapCreated$: of(true),
+    }
+  );
+
+@Component({
+  template: `
+    @if (show) {
+    <mgl-marker [lngLat]="[0,0]" #myMarker>
+      ...
+    </mgl-marker>
+    <mgl-popup [marker]="myMarker">
+        Hello from marker!
+    </mgl-popup>
+    }
+  `,
+  standalone: true,
+  imports: [MarkerComponent, PopupComponent]
+})
+class MarkerPopupTestComponent {
+    show = true;
+    public toggle() {
+        this.show = !this.show;
+    }
```

Comment:
I don't see this method being called anywhere in the test - is this the API? a marker component is shown/hidden using a public toggle method?

Reply:
Right, I ended up using the property. I'll remove this.